pkg/virtctl/adm/logverbosity/logverbosity.go

```diff
@@ -0,0 +1,304 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package logverbosity
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"k8s.io/client-go/tools/clientcmd"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+const (
+	LOG_VERBOSITY = "log-verbosity"
+)
+
+type Command struct {
+	clientConfig clientcmd.ClientConfig
+	command      string
+}
+
+var (
+	virtAPIFlag        bool
+	virtControllerFlag bool
+	virtHanderFlag     bool
+	virtLauncherFlag   bool
+	virtOperatorFlag   bool
```

Comment:
I know that this diverges a bit from the initial description of the issue, but maybe it would make more sense to use an int instead of a bool. Like:
```bash
virtctl adm log-verbosity --virt-handler 3 --virt-controller 2
virtctl adm log-verbosity --all 3
```

Reply:
As we discussed later in this PR, I also like this command format. I’ll change the code accordingly.